src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -391,7 +391,7 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 							stdout.Set ();
 					};
 					p.StartInfo = psi;
-					Console.WriteLine ($"{psi.FileName} {psi.Arguments}");
+					Console.WriteLine ($"{psi.FileName} {psi.Arguments} {string.Join (" ", File.ReadAllLines (responseFile))}");
```

Comment:
I think the response file would get zipped up in the test results. Did you need this to figure out a problem?

Reply:
I added this to check the JavaSdkDirectory value being used, but kept it because I thought some of the extra output was helpful and figured it likely has a negligible performance impact.

Reply:
I ended up backing this out as it's not necessary.